### PR TITLE
Add notes for 2016/06/07 meeting

### PR DESCRIPTION
**Pull request by beaugunderson**

### CODE REVIEW

**Comment by ore** [comment]
are the actual issue assigns on here? maybe i'm just blind

**Comment by ore** [comment]
@beaugunderson yeah that'd help

**Comment by beaugunderson** [comment]
@ore ok added :)

File: meetings/2016-06-07.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+- bryan: i've been doing some work on this but i'm open to others helping as
+  well
+- charlotte: what work have you been doing?
+- bryan: fleshing out the proposal (which is a bottleneck i think)
+- bryan: runbooks are a piece that will be helpful
+- charlotte: runbook?
+- bryan: 'if you encounter this, here are the steps to take'; a book of
+  scenarios so reactions can be predictable/repeatable
+- bryan: **TODO** i can continue to spearhead this one
+- bryan: a project manager would be helpful for these; someone to herd cats
```
Comment (nebrius) [inline meetings/2016-06-07.md]: This bullet point and down was more of a separate discussion, maybe call it out with a separate heading?

**Comment by beaugunderson** [comment]
@nebrius ok, moved it to a separate heading :)

**Comment by nebrius** [comment]
LGTM

File: meetings/2016-06-07.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+- jona: not a group that does just this; some individuals are
+- beau: i think docs does some of this
+- julián: current activity for evangelism is maintaining nodejs weekly;
+  individually attending events
+- julián: want to find out who's going to do this work (inclusivity?
+  evangelism?)
+- jona: maybe a WG is the right choice? we need to discuss into
+- charlotte: want to reach out to tracy as well
+- bryan: a coalition would be helpful for this
+- scott: tracy + docs + evangelism
```
Comment (scottgonzalez) [inline meetings/2016-06-07.md]: This wasn't me.

File: meetings/2016-06-07.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+- charlotte: and us maybe
+- bryan: do you want to work on this then?
+- scott: **TODO** yes
```
Comment (scottgonzalez) [inline meetings/2016-06-07.md]: This wasn't me either.

Comment (beaugunderson) [inline meetings/2016-06-07.md]: sorry scott! was typing by voice and was clearly a bit off today; i'll change these

**Pull request merged**